src/main/telemetry/jetiexbus.c

```diff
@@ -130,15 +130,21 @@ enum exSensors_e {
     EX_CURRENT,
     EX_ALTITUDE,
     EX_CAPACITY,
-    EX_FRAMES_LOST,                                                             // for debug only
-    EX_TIME_DIFF                                                                // for debug only
+    EX_POWER,
+    EX_ROLL_ANGLE,
+    EX_PITCH_ANGLE,
+    EX_HEADING
 };
 
 #define JETI_EX_SENSOR_COUNT (ARRAYLEN(jetiExSensors))
 
 static uint8_t jetiExBusTelemetryFrame[40];
 static uint8_t jetiExBusTransceiveState = EXBUS_TRANS_RX;
-static void sendJetiExBusTelemetry(uint8_t packetID);
+static uint8_t activeSensorList[JETI_EX_SENSOR_COUNT];
```

Comment:
(activeSensorList is only minor optimization, it may be replaced with `exSensorEnabled`, scanning all JETI_EX_SENSOR_COUNT)

Reply:
Is in progress...